packages/manager/modules/pci/src/projects/creating/index.scss

```
@@ -123,7 +123,7 @@ pci-project-creating {
 
     50% {
       stroke-dasharray: 120, 300;
-      stroke-dashoffset: -$d / 3;
+      stroke-dashoffset: calc(-#{$d} / 3);
```

```suggestion
      stroke-dashoffset: calc(-$d / 3);
```
Why need string encapsulation here as $d is a number ?